_data/trainings.yml

```diff
@@ -50,7 +50,13 @@
   description: For Java developers wanting to learn Functional Programming with one of the most exciting languages on the JVM
   page: /services/training/scala-for-java-developers
 
-- id: akka 
+- id: akka
   name: Scala Deep Dive with Akka
   description: For developers that want to deepen their Scala knowledge using its most popular toolkit, Akka
   page: /services/training/scala-deep-dive-with-akka
+
+- id: oo_design
+  name: Object Oriented Design Practise
+  author: Franziska Sauerwein
+  description: For experienced developers that want to practise creating object oriented designs.
```

Comment:
Maybe put a bit of an improvement notion here? "experienced developers who want to improve their design skills in object-oriented languages" ?

See Sandro's crafted design course description